src/playbooks/with_build/upgrade_vsds.yml

```diff
@@ -30,6 +30,7 @@
   tasks:
     - name: Stop replication process in the standby node
       command: /opt/vsd/bin/vsd-deactivate-replication-slave
+      changed_when: True
```

Comment:
Won't this mean the task will never fail? Shouldn't we be registering the output and checking for something?

Reply:
This means the task will always say that it has changed something. It's a switch between tasks marked yellow and tasks marked green. 
Ideally yes that would have been the best way. Finding out the return code of successful command execution and then setting the `changed_when` status